File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/Follow.java
package me.grishka.houseclub.api.methods;

import me.grishka.houseclub.api.BaseResponse;
import me.grishka.houseclub.api.ClubhouseAPIRequest;

public class Follow extends ClubhouseAPIRequest<BaseResponse> {
    public Follow(int userID) {
        super("POST", "follow", BaseResponse.class);
        requestBody = new Body(userID);
    }

    private static class Body {
        public int userId, source = 4;

        public Body(int userId) {
            this.userId = userId;
        }
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/Follow.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.BaseResponse
import me.grishka.houseclub.api.ClubhouseAPIRequest

class Follow(userID: Int) : ClubhouseAPIRequest<BaseResponse?>("POST", "follow", BaseResponse::class.java) {
    private class Body(var userId: Int) {
        var source = 4
    }

    init {
        requestBody = Body(userID)
    }
}